grantnav/frontend/templates/search.html

```diff
@@ -88,7 +88,7 @@ <h4 class="smaller-top-margin-heading"> Current Filters </h4>
     {% endif %}
 
     <div class="col-xs-3 filters">
-      <h3 class="indent"> Filter By </h3>
+      <h3 class="indent"> Filter By <a class="filter_by_help_link" href="http://grantnav.threesixtygiving.org/help#filters"><sup><img src="{% static 'images/icon-help-50.png'%}" width="15" height="15"></sup></a></h3>
```

Comment:
Please can we make this a relative link? 

```suggestion
      <h3 class="indent"> Filter By <a class="filter_by_help_link" href="/help#filters"><sup><img src="{% static 'images/icon-help-50.png'%}" width="15" height="15"></sup></a></h3>
```

Reply:
@robredpath do you know how to test the link with a relative link?

Reply:
I have fix that now.